### PR TITLE
 Issue #6320: add remove conditionals mutator for coding package 

### PR DESCRIPTION
**Pull request by rnveach**
Issue https://github.com/checkstyle/checkstyle/issues/6320

Added new mutator and starting list.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2249,7 +2248,7 @@
                 <param>*.Input*</param>
               </excludedTestClasses>
               <coverageThreshold>100</coverageThreshold>
-              <mutationThreshold>100</mutationThreshold>
+              <mutationThreshold>97</mutationThreshold>
```
Comment (romani) [inline pom.xml:2251]: If we reduce it, it might cause leak in other mutators

**Review by romani — CHANGES_REQUESTED** [review]
Item:

File: pom.xml (hunk 1 above)
Comment (rnveach) [reply pom.xml]: It is being reduced to 99%. List is being created of outstanding items.

**Review by romani — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (romani) [reply pom.xml]: while we have validation in shell script we are safe.

**Review by pbludov — APPROVED** [review]

**Pull request merged**